system-addon/content-src/components/Search/Search.jsx

```diff
@@ -49,10 +59,11 @@ class Search extends React.Component {
       </label>
       <input
         id="newtab-search-text"
+        ref={input => { this.input = input; }}
         maxLength="256"
         placeholder={this.props.intl.formatMessage({id: "search_web_placeholder"})}
-        ref={this.onInputMount}
         title={this.props.intl.formatMessage({id: "search_web_placeholder"})}
+        onFocus={this.controller ? null : this.onFocus}
```

Comment:
Will we ever `render` when `controller` already exists? Seems like `onFocus` will be called on focus each time anyway and there's guards to prevent reinitializing.